aspnet/fundamentals/localization/sample/Controllers/ZebraController.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
```

Comment:
Why is this named Zebra? Perhaps "InfoController.cs" would be a more helpful name (With the associated changes to all other Zebra related things of course).